core/src/test/java/org/apache/spark/Java8RDDAPISuite.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package test.org.apache.spark.java8;
```

Comment:
Sounds good. I was matching `JavaAPISuite` which isn't in test scope, but I agree with you. (I could move `JavaAPISuite' too?)